zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -10869,6 +10869,24 @@ components:
         errorMessage:
           description: the error message from the engine in case of a failed operation.
           type: string
+    ProcessInstanceCancellationBatchOperationRequest:
+      type: object
+      description: |
+        The process instance filter that defines which process instances should be canceled.
+      properties:
+        filter:
+          $ref: "#/components/schemas/ProcessInstanceFilter"
+      required:
+        - filter
```

Comment:
🔧 This prevents sending an empty request body (i.e., `{}`). But the controller doesn't enforce this, actually. So, this is not correct.